Microsoft.Toolkit.Uwp.UI.Controls.Design/TabbedCommandBarMetadata.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Windows.Design.Metadata;
+using Microsoft.Windows.Design.PropertyEditing;
+using System.ComponentModel;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Design
```

Comment:
We're moving to the new format in #3318. I don't think there's a way for you to pre-emptively move there though until that's merged.

@Nirmal4G correct?

Reply:
Yes